app/jobs/reports/verification_errors_report.rb

```diff
@@ -0,0 +1,67 @@
+require 'identity/hostdata'
+require 'csv'
+
+module Reports
+  class VerificationErrorsReport < BaseReport
+    REPORT_NAME = 'verification-errors-report'.freeze
+
+    include GoodJob::ActiveJobExtensions::Concurrency
+
+    good_job_control_concurrency_with(
+      total_limit: 1,
+      key: -> { "#{REPORT_NAME}-#{arguments.first}" },
+    )
+
+    def perform(date)
+      csv_reports = []
+      configs = IdentityConfig.store.verification_errors_report_configs
+      configs.each do |report_hash|
+        name = report_hash['name']
+        issuers = report_hash['issuers']
+        report = verification_errors_data_for_issuers(date, name, issuers)
+        csv_reports << report
+      end
+      csv_reports
+    end
+
+    private
+
+    def verification_errors_data_for_issuers(date, report_name, issuers)
+      csv = CSV.new('', row_sep: "\r\n")
+      csv << %w[uuid welcome_view_at error_code]
+      issuers.each do |issuer|
+        transaction_with_timeout do
+          rows = ::VerificationErrorsReport.call(
+            issuer,
+            (date.beginning_of_day - 1).beginning_of_day,
+            date.beginning_of_day,
+          )
+          rows.each do |row|
+            csv << [row['uuid'], row['welcome_view_at'].iso8601, ial2_error_code(row)]
+          end
+        end
+      end
+      data = csv.string
+      save_report(report_name, data, extension: 'csv')
```

Comment:
this will save each issuer's `.latest` separately, right?

Reply:
yes